### PR TITLE
Edit README

### PR DESCRIPTION
**Pull request by Keshraf**
### Use:
- A gesture to thank the contributors for their efforts
- Increases engagement with other open-source enthusiasts
- Increases credibility as an open-source project
- Looks good in the README as well 😉

### CODE REVIEW

**Comment by Keshraf** [comment]
Hello Project Maintainers,
The feature added by me uses a dynamic link. Therefore, the images would be increased on their own depending on the contributors of the project.

File: README.md [hunk 1]
```diff
@@ -43,32 +43,34 @@ An app for all those who care about their health.
...
 <td align="center"><a href="https://github.com/ghulamyazdani"><img src="https://avatars.githubusercontent.com/u/55938346?v=4" width=150px height=150px /></a></br> <h4 style="color:white;">Ghulamyazdani</h4>
 
-     
 </tr>
 </table>
 <br>
 
+<br>
+
+<h1 align=left> 🙌 Contributors</h1>
```
Comment (ghulamyazdani) [inline README.md:71]: It would be better if you  center align it!!

**Review by ghulamyazdani — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Keshraf) [reply README.md]: Done

**Comment by ghulamyazdani** [comment]
Thanks for your valuable contribution!

**Pull request merged**